src/ServiceControl.Persistence.Tests.RavenDb/TestPersistenceImpl.cs

```diff
@@ -70,7 +69,7 @@ public override void BlockToInspectDatabase()
                 return;
             }
 
-            var url = $"http://localhost:{settings.DatabaseMaintenancePort}/studio/index.html#databases/documents?&database=%3Csystem%3E";
+            var url = $"http://localhost:{(Settings as RavenDBPersisterSettings).DatabaseMaintenancePort}/studio/index.html#databases/documents?&database=%3Csystem%3E";
```

Comment:
```suggestion
            var databaseMaintenanceUrl = ((RavenDBPersisterSettings)Settings).DatabaseMaintenanceUrl
            var url = databaseMaintenanceUrl + $"/studio/index.html#databases/documents?&database=%3Csystem%3E";
```

Reply:
This is an unrelated change too

Reply:
Your change was doing an `as` which could resolve to `null`. This change uses a cast which seems to me would be a better fit as the instance should always be that type.